Description: Fix for #807657 -- missing include "config.h"
Author: Tobias Frost <tobi@frost.de>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=807657
Last-Update: 2016-01-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/compat.h
+++ b/src/compat.h
@@ -1,6 +1,8 @@
 // basic headers
 #define _GNU_SOURCE /* really just for strndup */
 
+#include "config.h"
+
 #ifdef HAVE_STDBOOL_H
 # include <stdbool.h>
 #else
@@ -17,6 +19,7 @@
 # define __bool_true_false_are_defined 1
 #endif
 
+
 #include <stdio.h>
 
 #ifdef HAVE_STDLIB_H 
